packages/block-editor/src/components/index.native.js

```diff
@@ -91,6 +91,7 @@ export { default as __unstableEditorStyles } from './editor-styles';
 export { default as Inserter } from './inserter';
 export { useBlockProps } from './block-list/use-block-props';
 export { default as FloatingToolbar } from './floating-toolbar';
+export { default as BlockFallbackWebVersion } from './block-fallback-web-version';
```

Comment:
Do we need to export this component or should it be considered internal/private?

Reply:
We don't need to export it in this PR because it's only used within the same package. However, my plan is to use this component also in the Missing block (i.e. the one used to display unsupported blocks). This block is in a different package, so we'd need to export it for that case. That said, I agree that since it's not needed in these changes, we can remove the export and incorporate it in the future when needed 👍 .

Reply:
Solved in https://github.com/WordPress/gutenberg/pull/54382/commits/4ae212b19579900d790496b6ac0db30594551752.

Reply:
> [...] However, my plan is to use this component also in [the Missing block](https://github.com/WordPress/gutenberg/blob/45f5608443d939dac1e21b3bedd0a2662992711b/packages/block-library/src/missing/edit.native.js#L295) (i.e. the one used to display unsupported blocks). This block is in a different package, so we'd need to export it for that case. [...]

@dcalhoun In relation to this comment, I'd like to share that I opened [this PR](https://github.com/WordPress/gutenberg/pull/55133) to use the new `UnsupportedBlockDetails` component in the Missing block. Let me know if you could take a look, thanks 🙇 !